ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -65,25 +243,125 @@ class SignalSpec: QuickSpec {
 		}
```

Comment:
I think the easiest way to test this would be to capture some object in your observer callbacks, then make sure that object gets released (as a proxy for identifying when the observer itself is released).